Text/PresidioPIIAnonymization/notebooks/PII Anonymization.ipynb

```diff
@@ -0,0 +1,381 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from azure.core.credentials import AzureKeyCredential\n",
+    "from azure.search.documents import SearchClient\n",
+    "from IPython.display import Markdown, display\n",
+    "from requests import post, put\n",
+    "\n",
+    "def printmd(string):\n",
+    "    display(Markdown(string))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test our text on our local running API"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "API_KEY = \"[YourSecretKeyCanBeAnything]\"\n", # Set the KEY value you deployed your Web App with [YourSecretKeyCanBeAnything]\n",
+    "URL_LOCAL = f\"http://0.0.0.0:5000/api/extraction\"\n",
```

Comment:
There seems to be a formatting issue here, I think a missing quote before the '#'.

Reply:
fixed.